From: Stefano Rivera <stefanor@debian.org>
Date: Thu, 8 Oct 2015 11:02:04 -0700
Subject: Use Debian's libstemmer.

Dynamically Link against libstemmer rather than using included
libstemmer_c sources.

Forwarded: not-needed
Last-Update: 2020-07-14
---
 setup.py | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/setup.py b/setup.py
index b83cbe5..76d4c70 100644
--- a/setup.py
+++ b/setup.py
@@ -108,9 +108,10 @@ class LibrarySourceCode:
             expected_sha256=checksum or self.DEFAULT_CHECKSUM)
 
 
-LIBRARY_SOURCE_CODE = LibrarySourceCode()
-if not LIBRARY_SOURCE_CODE.is_present_on_disk():
-    LIBRARY_SOURCE_CODE.download()
+# Debian: Use packaged libstemmer
+#LIBRARY_SOURCE_CODE = LibrarySourceCode()
+#if not LIBRARY_SOURCE_CODE.is_present_on_disk():
+#    LIBRARY_SOURCE_CODE.download()
 
 
 class BootstrapCommand(Command):
@@ -191,9 +192,8 @@ setup(name='PyStemmer',
       ext_modules=[
         Extension(
             'Stemmer',
-            ['src/Stemmer.pyx'] + list(LIBRARY_SOURCE_CODE.source_code_paths()),
-            include_dirs=LIBRARY_SOURCE_CODE.include_directories
+            ['src/Stemmer.pyx'],
+            libraries=['stemmer']
         )
       ],
-      cmdclass={'bootstrap': BootstrapCommand}
       )
